README.md

```diff
@@ -68,27 +70,60 @@ can only be specified when configuring the monitoring or logging agents.
 For more information, please see [Monitoring third-party
 applications](https://cloud.google.com/monitoring/agent/plugins).
 
-Example Playbook
+Example Playbooks
 ----------------
 
 ```
-# Example
+# Installing the Monitoring and Logging agents
+- hosts: all
+  become: true
+  roles:
+    - role: google_cloud_ops_agents
+      vars:
+        agent_type: monitoring
+
+    - role: google_cloud_ops_agents
+      vars:
+        agent_type: logging
+```
+```
+# Installing the Monitoring and Logging agents with custom configurations
 - hosts: all
   become: true
   roles:
     - role: google_cloud_ops_agents
       vars:
         agent_type: monitoring
-        package_state: present
         version: latest
         main_config_file: collectd.conf
```

Comment:
Regarding the sample values of `main_config_file` and `additional_config_dir`, can we keep them under the same pattern across different agents? e.g.

XX_agent.conf
XX_agent_dir/

where XX is logging, monitoring, ops?

Reply:
Done.